raiden/network/proxies/secret_registry.py

```diff
@@ -71,6 +75,22 @@ def __init__(self, jsonrpc_client, secret_registry_address, contract_manager: Co
     def register_secret(self, secret: Secret):
         self.register_secret_batch([secret])
 
+    def register_secret_light(self, signed_tx: SignedTransaction):
+        try:
+            log.debug("register_secret_light called")
+            transaction_hash = self.proxy.broadcast_signed_transaction(signed_tx)
+            self.client.poll(transaction_hash)
+            receipt_or_none = check_transaction_threw(self.client, transaction_hash)
+            if receipt_or_none:
```

Comment:
i know this is taken from somewhere else, but i'd rather call this variable `receipt` and then just check if it's not `None`.

`receipt_or_none` is such a confusing name for me. we've discussed this with @JONAF2103 before